action.yml

```diff
@@ -126,6 +220,29 @@ runs:
         path: .coverage-output.final
         GITHUB_TOKEN: ${{ inputs.token }}
 
+    - name: Commit XML report to ${{ inputs.diff-storage }}
+      if: >
+        contains(inputs.diff, 'true')
+      working-directory: ${{ inputs.diff-storage }}
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Actions"
+        git switch ${{ inputs.diff-storage }}
+        git pull origin ${{ inputs.diff-storage }}
+        mv ../coverage-report-action/coverage.xml ./data/${{ env.diff_storage_branch }}/coverage.xml
```

Comment:
Looks like this results in an error, per the tests on this repo.

Reply:
Fixed, this was caused by the action not expecting coverage report to be in a differently named file and possibly in a subdirectory like here:
https://github.com/insightsengineering/coverage-action/blob/main/.github/workflows/test.yaml#L55

Two test attempts, both successful:
https://github.com/insightsengineering/coverage-action/actions/runs/2780956007/attempts/1
https://github.com/insightsengineering/coverage-action/actions/runs/2780956007

Reply:
I also re-ran coverage action on the test repo:
https://github.com/walkowif/tern/pull/7
https://github.com/walkowif/tern/pull/8